src/lib/handlebars-converter/handlebars-helpers.spec.js

```diff
@@ -253,14 +280,22 @@ describe('Handlebars helpers', function () {
         assert.strictEqual('123-45-67', getHelper('addHyphensSSN').func('123-45-67'));
     });
 
-    it('addHyphensDate adds hyphens when passed 8 digits', function () {
-        assert.strictEqual('2001-01-02', getHelper('addHyphensDate').func('20010102'));
+    it('addHyphensDate adds hyphens when passed 4 digits', function() {
+        assert.strictEqual('2001', getHelper('addHyphensDate').func('2001'));
     });
 
-    it('addHyphensDate leaves input unchanged when not 8 digits', function () {
-        assert.strictEqual('123', getHelper('addHyphensDate').func('123'));
-        assert.strictEqual('111111111111', getHelper('addHyphensDate').func('111111111111'));
-        assert.strictEqual('2001-01-', getHelper('addHyphensDate').func('2001-01-'));
+    it('addHyphensDate adds hyphens when passed 6 digits', function() {
+        assert.strictEqual('2001-01', getHelper('addHyphensDate').func('200101'));
+        assert.strictEqual('2001-06', getHelper('addHyphensDate').func('200106'));
+        assert.strictEqual('2001-12', getHelper('addHyphensDate').func('200112'));
+    });
+
+    it('addHyphensDate adds hyphens when passed more than 8 digits', function() {
+        assert.strictEqual('2001-12-01', getHelper('addHyphensDate').func('20011201'));
+        assert.strictEqual('2001-12-31', getHelper('addHyphensDate').func('20011231'));
+        assert.strictEqual('2001-01-31', getHelper('addHyphensDate').func('20010131'));
+        assert.strictEqual('2001-01-31', getHelper('addHyphensDate').func('2001013100'));
+        assert.strictEqual('2001-01-31', getHelper('addHyphensDate').func('200101310000'));
```

Comment:
Can we move these 3 tests to above group test (along with formatAsDateTime)?

Reply:
Yes, I also think they should be put on group test. I just follow the previous position. If it doesn't affect others, I move them to group test